todoman/model.py

```diff
@@ -47,7 +61,8 @@ class Todo:
 
     _localtimezone = tzlocal()
 
-    def __init__(self, todo=None, filename=None):
+    def __init__(self, todo=None, filename=None, mtime=None, safe=False,
```

Comment:
I'm unsure if I'm correctly understanding the context, but I think you meant to write `save` instead of `safe`.